engines/net.c

```diff
@@ -960,6 +1004,11 @@ static int fio_netio_send_open(struct thread_data *td, struct fio_file *f)
 	if (is_ipv6(o)) {
 		len = sizeof(nd->addr6);
 		to = (struct sockaddr *) &nd->addr6;
+#ifdef CONFIG_VSOCK
+	} else if (is_vsock(o)) {
```

Comment:
This uses and ifdef, the one right above it does not. Please be consistent.

Reply:
The inconsistency is due to the fact that some of the `if` branches -  like this one - use vsock defintions (like `addr_vm`) and others - like for instance [the one at line 934](https://github.com/MPinna/fio/blob/dc6667881019faf2042ac9b83996dc62b56cb863/engines/net.c#L934) - don't